### ISSUE TITLE
Stops working with Android 14

### ISSUE THREAD

**Issue by snapple42** [AndreyPavlenko/Fermata#304]
I upgraded to Android 14, and when I click on the icon, the screen sits there for a second and then crashes. The " start menu" disappears and and only has the screen layout button there to go from full screen to split screen.

Tried  to reinstall on king installer, and AAAD...

Suspect they did something that prevents it from launching the new version of Android.

**Comment by snapple42** [comment]
Saw it was duplicate

**Comment by scheuerer** [comment]
I had the same. Deinstalled AA to stock version, upgraded to latest version from the store, and it is working again without touching something in the 1.95 from Fermata…

**Comment by tarasov503** [comment]
> I had the same. Deinstalled AA to stock version, upgraded to latest version from the store, and it is working again without touching something in the 1.95 from Fermata…

On a pixel?

**Comment by scheuerer** [comment]
On an Samsungs Galaxy S23 ultra

**Comment by ShelbyGT35** [comment]
Made A14 Update for S23U and Fermata Auto isn't working anymore!!

**Comment by scheuerer** [comment]
As said before, after upgrade to A14, deinstalling AA to stock version and update to latest google store version, it works as before. Look here direct after A14 update, and after fixing this issue without reinstalling Fermata…

https://www.dropbox.com/scl/fo/vr2tqu0i5499edrgrz9bw/h?rlkey=3cl9eisfy0754hed3dswlvixb&dl=0

**Comment by ShelbyGT35** [comment]
Thanks a lot!!

But it's not working for me.

Fermata Auto 1.9.4 (all rights)
AA 10.6.634044 (developer status, all rights)
S23 Ultra Android 14 (One UI 6)


⁣Kind regard


Thomas

**Comment by Dannytgt69** [comment]
It's not working for me neither, i've tried with a Pixel 7 pro, and doesn't surprise me at all, but with a s23 ultra at least the app it's launched un the car stereo, but then ir crashes, when I enter the app in the phone i got the " this app there's not compatible with this android versión".

**Comment by snapple42** [comment]
I just installed over using King Installer, and it worked for me...

**Comment by aross6** [comment]
> I just installed over using King Installer, and it worked for me... I have a pixel 6a

Are you on Android 14?

**Comment by snapple42** [comment]
Yup... current to Android version: 14, Android Security Update: October 5, 20203

**Comment by omelyanchuk85** [comment]
> Ага... текущая версия Android: 14, обновление безопасности Android: 5 октября 2020 г. 3

Можете подробнее описать порядок действий? Какой версии использовали kinginstaller? Версия АА?
У меня pixel 5, Android 14, AA 10.9.134544.
Не получается установить так, чтобы Fermata Auto отображалась в АА

**Comment by nikjev906** [comment]
My samsung a54 update to Android 14 , fermata auto stop working and android auto doesn't have the app so I can't have it in my car

**Comment by junak007** [comment]
Hi folks, i havw tried this several times. Enable developer mode on AA after, go to 3 dots on top settings and make sure unknown sources has checkmark in it, after taht use king installer and install app that you download from below link.

https://mega.nz/file/GV0AGYLD#P6GKGBU5R30sBt3_Nshw3fahLDyuJ5Qt8zEMoi3PXlk

Name of the installed app is not Fermata Auto but its a same app as fermata auto with all same functions and you can get video, iptv and youtube to play on your AA, i found this app like that and i didnt do anything to it.  Now when i open AA and open customize app screen i get this app showing up and it should work in car as well, will try in morning. I hope you can check yourself also and use it. 

#Andrey Pavlenko , can you check this app and see what its done to it to work and maybe add it to your original Fermata Auto.

**Comment by nikjev906** [comment]
I'll try it thank you

Στις Σάβ 16 Δεκ 2023, 22:13 ο χρήστης junak007 ***@***.***>
έγραψε:

**Comment by danchitu866** [comment]
> Hi folks, i havw tried this several times. Enable developer mode on AA after, go to 3 dots on top settings and make sure unknown sources has checkmark in it, after taht use king installer and install app that you download from below link.
> 
> https://mega.nz/file/GV0AGYLD#P6GKGBU5R30sBt3_Nshw3fahLDyuJ5Qt8zEMoi3PXlk
> 
> Name of the installed app is not Fermata Auto but its a same app as fermata auto with all same functions and you can get video, iptv and youtube to play on your AA, i found this app like that and i didnt do anything to it. Now when i open AA and open customize app screen i get this app showing up and it should work in car as well, will try in morning. I hope you can check yourself also and use it.
> 
> #Andrey Pavlenko , can you check this app and see what its done to it to work and maybe add it to your original Fermata Auto.

I confirm it's working, but don't know how safe it's the app...

**Comment by rraghava10** [comment]
In my case it ok s not working same as original Fermata Auto,yt is not visited ble in car

**Comment by AndreyPavlenko** [comment]
> Hi folks, i havw tried this several times. Enable developer mode on AA after, go to 3 dots on top settings and make sure unknown sources has checkmark in it, after taht use king installer and install app that you download from below link.
> 
> https://mega.nz/file/GV0AGYLD#P6GKGBU5R30sBt3_Nshw3fahLDyuJ5Qt8zEMoi3PXlk
> 
> Name of the installed app is not Fermata Auto but its a same app as fermata auto with all same functions and you can get video, iptv and youtube to play on your AA, i found this app like that and i didnt do anything to it. Now when i open AA and open customize app screen i get this app showing up and it should work in car as well, will try in morning. I hope you can check yourself also and use it.
> 
> #Andrey Pavlenko , can you check this app and see what its done to it to work and maybe add it to your original Fermata Auto.

It works on A14?

**Comment by ShelbyGT35** [comment]
Just tried it in the car. I get only displayed to choose Smart TV in AA Menu but without playing function.

**Comment by alpayyildirim** [comment]
I also tried it on a14, it comes into AA but it doesn't play.

**Comment by SpideySparks** [comment]
> > Hi folks, i have tried this several times. Enable developer mode on AA after, go to 3 dots on top settings and make sure unknown sources has checkmark in it, after taht use king installer and install app that you download from below link.
> > https://mega.nz/file/GV0AGYLD#P6GKGBU5R30sBt3_Nshw3fahLDyuJ5Qt8zEMoi3PXlk
> > Name of the installed app is not Fermata Auto but its a same app as fermata auto with all same functions and you can get video, iptv and youtube to play on your AA, i found this app like that and i didnt do anything to it. Now when i open AA and open customize app screen i get this app showing up and it should work in car as well, will try in morning. I hope you can check yourself also and use it.
> > #Andrey Pavlenko , can you check this app and see what its done to it to work and maybe add it to your original Fermata Auto.
> 
> Do you mean it works on A14?

File was removed from the link!

**Comment by agostinho54** [comment]
Qual e o link ?

**Comment by mousesport** [comment]
Give me link
